config/pipeline.yaml

```diff
@@ -212,14 +212,6 @@ runtimes:
       callback:
         token: kernelci-api-token-staging
 
-  # ToDo: avoid creating a separate Runtime entry
-  # https://github.com/kernelci/kernelci-core/issues/2088
-  lava-collabora-early-access:
-    <<: *lava-collabora-staging
-    notify:
-      callback:
-        token: kernelci-api-token-early-access
-
```

Comment:
I couldn't find any job using this lab.
Is it safe to drop this lab? @nuclearcat

Reply:
seems yes, thanks for cleanup!